src/js/components/Diagram/StyledDiagram.js

```diff
@@ -1,12 +1,90 @@
-import styled from 'styled-components';
+import styled, { css, keyframes } from 'styled-components';
 
 import { defaultProps } from '../../default-props';
 
+const PULSE_SIZES = {
+  xsmall: 1.001,
+  small: 1.01,
+  medium: 1.1,
+  large: 1.5,
+  xlarge: 2,
+};
```

Comment:
Are these the same sizes you'll see on Box's pulse? If the code is repeated we might want to add it to utils and reuse it.

Reply:
Sounds like a good plan! Thanks!

Reply:
After testing the functionalities moving many of the overlapping helper functions to utils, I think it's safe to resolve this one 😄